Taskfile.yml

```diff
@@ -19,6 +19,11 @@ includes:
       PYTHON_VERSION: 3.11
       VERSION: '{{.VERSION}}'
 
+  base:
```

Comment:
A divergence from our prior language-specific abstractions; init in the past was entirely independent of the `goat/` but since `init-submodules` happens before `init-install-tools` it shouldn't be a problem.  I think